### ISSUE TITLE
Expose service names available in tnsnames.ora

### ISSUE THREAD

**Issue by mac-vtl** [oracle/python-oracledb#313]
1. Describe your new request in detail

`python-oracledb` internally parses the content of the [`tnsnames.ora`](https://docs.oracle.com/en/database/oracle/oracle-database/18/netrf/local-naming-parameters-in-tnsnames-ora-file.html#GUID-1DECC117-BB0F-4BBE-9C88-479ECB202DBB) file [here](https://github.com/oracle/python-oracledb/blob/40d2b9789322ec0eefe43ccf6e86a25e98faf71b/src/oracledb/impl/base/connect_params.pyx#L1005) but does not provide or expose a way to list the available service names parsed.

We have use cases where we need to switch between database services. It would be logical to propose the list of available services from the list of services parsed by python-oracledb.

Something like below or similar would work:

```py
from oracledb.connect_params import ConnectParams
params = ConnectParamsImpl()
service_names = params.available_service_names()
# do something with service_names
print(service_names)
```
Is that something feasible? What are your thoughts?

2. Give supporting information about tools and operating systems.  Give relevant product version numbers

```py
import oracledb
import sys
import platform

print("oracledb.__version__:", oracledb.__version__)
print("platform.platform:", platform.platform())
print("sys.maxsize > 2**32:", sys.maxsize > 2**32)
print("platform.python_version:", platform.python_version())
```
```sh
oracledb.__version__: 2.1.0
platform.platform: Windows-10-10.0.19044-SP0
sys.maxsize > 2**32: True
platform.python_version: 3.11.5
```

**Comment by anthony-tuininga** [comment]
Yes, that would be possible and shouldn't be too much effort, either.

**Comment by anthony-tuininga** [comment]
I have added the function get_network_service_names() which does what you request. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it works for you, too.

**Comment by mac-vtl** [comment]
not currently setup to build from source but will try to get setup an provide feedback asap. thanks for the quick turnaround.

**Comment by mac-vtl** [comment]
I can confirm it works as expected. tyvm.

**Comment by anthony-tuininga** [comment]
This was included in version 2.2.0 which was just released.
